frontend/src/components/display-products/Products.tsx

```diff
@@ -0,0 +1,122 @@
+// React
+import React from "react";
+import { useNavigate } from "react-router";
+// Types
+import { productProps } from "./displayProductsInterface";
+// Material Ui
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+import { CardActionArea } from "@mui/material";
+import Box from "@mui/material/Box";
+// Components
+import RatingComponent from "./Rating";
+import AddToCartButton from "../AddToCartButton";
+import ProductQuantity from "./ProductQuantity";
+
+const styles = {
+    title: { textAlign: "center", marginTop: "90px" },
+    container: {
+        display: "flex",
+        flexWrap: "wrap",
+        justifyContent: "space-around",
+        maxWidth: "1250px",
+        margin: "0 auto",
+        marginTop: "50px",
+    },
+    card: {
+        maxWidth: 500,
+        marginBottom: "60px",
+        width: "400px",
+    },
+    cardMedia: {
+        height: "390px",
+        width: "100%",
+        objectFit: "contain",
+    },
+    price: {
+        marginBottom: 0,
+    },
+    description: {
+        marginBottom: "16px",
+    },
+};
+
+const Products: React.FC<productProps> = ({ data }) => {
+    const navigate = useNavigate();
+    const { container, card, cardMedia, price, description, title } = styles;
+    return (
+        <Box id="products-container">
+            <Typography sx={ title } gutterBottom variant="h4" component="div">
+                Featured Products
+            </Typography>
+            <Box sx={ container }>
+                { data
+                    ? data?.map((product) => {
```

Comment:
`data.map((product) => {`
the ? operator is redundant here

Reply:
Ignore this file, it will gone once i merge it with main, this has already been changed in main.